bin/cli.js

```diff
@@ -112,11 +112,9 @@ prettyCLI
         } else {
             gscan.check(argv.themePath, cliOptions)
                 .then(theme => outputResults(theme, cliOptions))
-                .catch(function ENOTDIRPredicate(err) {
-                    return err.code === 'ENOTDIR';
-                }, function (err) {
-                    ui.log(err.message);
-                    ui.log('Did you mean to add the -z flag to read a zip file?');
+                .catch((err) => {
+                    ui.log(`\n${chalk.red(err.message)}`);
```

Comment:
Hm. Would be good to check the style consistency here imo. We don't seem to log errors this way anywhere else in the CLI. The catch block above for example just uses 'ui.log'. Also, there's the `levels` configuration up above that configures the "error" [log to be red](https://github.com/TryGhost/gscan/pull/466/files#diff-ca900686fca4680d4692bf587dc5da1322879c344688c187b5511cda27902c18R14). 

Do we only log "theme" errors as red elsewhere?